src/main.rs

```diff
@@ -6,27 +6,14 @@ use appflowy_cloud::telemetry::init_subscriber;
 async fn main() -> anyhow::Result<()> {
   let level = std::env::var("RUST_LOG").unwrap_or("info".to_string());
   println!("AppFlowy Cloud with RUST_LOG={}", level);
-  let mut filters = vec![];
-  filters.push(format!("actix_web={}", level));
-  filters.push(format!("collab={}", level));
-  filters.push(format!("collab_sync={}", level));
-  filters.push(format!("appflowy_cloud={}", level));
-  filters.push(format!("collab_plugins={}", level));
-  filters.push(format!("realtime={}", level));
-  filters.push(format!("database={}", level));
-  filters.push(format!("storage={}", level));
-  filters.push(format!("gotrue={}", level));
-  filters.push(format!("appflowy_collaborate={}", level));
-  filters.push(format!("appflowy_ai_client={}", level));
-  filters.push(format!("indexer={}", level));
```

Comment:
We need to be wary how these changes will affect test/beta deployments.

Reply:
In production environment, we have defined RUST_LOG=info, and the same applies to self hosters too, for the default deployment. We can adjust RUST_LOG as necessary.